src/mono/mono/mini/simd-intrinsics-netcore.c

```diff
@@ -2181,7 +2181,9 @@ MONO_EMPTY_SOURCE_FILE (simd_intrinsics_netcore);
 void
 ves_icall_System_Runtime_Intrinsics_X86_X86Base___cpuidex (int abcd[4], int function_id, int subfunction_id)
 {
+#ifndef MONO_CROSS_COMPILE
```

Comment:
How does this affect the autotools builds? They'll never define `MONO_CROSS_COMPILE`, right?

Reply:
This will only affect cross compilers.